game.cpp

```diff
@@ -761,6 +762,23 @@ void place_nth_food_randomly(game &g, const int &n)
   g.get_food()[n].place_randomly(g.get_rng(), {get_min_x(g), get_min_y(g)}, {get_max_x(g), get_max_y(g)});
 }
 
+coordinate get_nth_shelter_position(const game &g, const int &n)
+{
+  return g.get_shelters()[n].get_position();
```

Comment:
Nice first setup. To be examplary, don't forget that `i` needs to be positive, for `g.get_shelters()[n]` to make sense. Hence, adding `assert(n >= 0);` give a nice error message during debug.

Reply:
Yay!